src/stcal/alignment/resample_utils.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import numpy as np
+from stcal.alignment import util
+from gwcs.wcstools import grid_from_bounding_box
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def calc_pixmap(in_wcs, out_wcs, shape=None):
+    """Return a pixel grid map from input frame to output frame
+
+    Parameters
+    ----------
+    in_wcs: `~astropy.wcs.WCS`
+        Input WCS objects or transforms.
+    out_wcs: `~astropy.wcs.WCS` or `~gwcs.wcs.WCS`
```

Comment:
nitpick: the convention for the parameters is to have a space between the name and `:`, as in `in_wcs :`
This needs to be fixed in the entire module.